sceneview/src/main/java/io/github/sceneview/node/Node.kt

```diff
@@ -688,24 +688,69 @@ open class Node(
         }
     }
 
-    // TODO
-//
-//    /**
-//     * ### Rotates the node to face a point in world space
-//     *
-//     * World-space up (0, 1, 0) will be used to determine the orientation of the node around the
-//     * direction.
-//     *
-//     * @param position The position to look at in world space
-//     * @param up The up direction will determine the orientation of the node around the direction
-//     */
-//    fun lookAt(position: Position, up: Direction = Direction(y=1.0f), smooth: Boolean = false) {
-//        if(smooth) {
-//            smooth(quaternion = rotation(lookAt(this.worldPosition, position, up)).toQuaternion())
-//        } else {
-//            transform(rotation = rotation(lookAt(this.worldPosition, position, up)).toQuaternion())
-//        }
-//    }
+    /**
+     * ### Rotates the node to face a point in world-space
+     *
+     * @param targetPosition The target position to look at in world space
+     * @param upDirection The up direction will determine the orientation of the node around the direction
```

Comment:
It is not very clear in my opinion. Can we write `The up direction determines where the top of the node will point after the rotation is applied`?

Reply:
I'm not sure if that's what the upDirection is. `The up vector of the rotation will only match the worldUp vector if the forward direction is perpendicular to worldUp.`
https://docs.unity3d.com/ScriptReference/Transform.LookAt.html
http://glm.g-truc.net/0.9.4/api/a00151.html#gae2dca3785b6d5796e876114af58a60a1

Reply:
Yes, it will rotate the y+ axis around the look direction to minimize the angle between the y+ axis and the up direction.

Reply:
Probably, we can just add `look` before `direction`.

Reply:
Done!